server/paths/api.ts

```diff
@@ -28,9 +28,9 @@ export default {
     index: submissionsPath,
     create: submissionsPath,
     show: singleSubmissionPath,
-    applicationNotes: {
-      create: singleSubmissionPath.path('notes'),
-    },
+    // applicationNotes: {
```

Comment:
looks like a bit of leftover commented out code

Reply:
Removed now.